l10n_mx_edi_addenda_mabe/models/account_move.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    # New Fields
+    sale_id = fields.Many2one("sale.order", readonly=True)
+    reference_1 = fields.Char(string="Referencia 1", default="NA")
+    reference_2 = fields.Char(string="Referencia 2", default="NA")
+    importe_con_letra = fields.Char(string="Importe con letra")
```

Comment:
@santiagordz Can you put the field name and label in english? We can export those later to translate them to spanish.